integration-test/example.test.js

```diff
@@ -0,0 +1,85 @@
+import app from '../example/devServer.js'
+import { launch } from 'puppeteer'
+import { toMatchImageSnapshot } from 'jest-image-snapshot'
+
+expect.extend({ toMatchImageSnapshot })
+
+const PORT = 9000
+
+const urlWhitelist = [
+  new RegExp(`http:\/\/localhost:${PORT}.*`),
+  /https:\/\/unpkg\.com\/react@[\d\.]+\/dist\/react\.min\.js/,
+  /https:\/\/unpkg\.com\/react-dom@[\d\.]+\/dist\/react-dom\.min\.js/,
+  /https:\/\/unpkg\.com\/react-dom@[\d\.]+\/dist\/react-dom-server\.min\.js/,
+  /https:\/\/cdnjs\.cloudflare\.com\/ajax\/libs\/babel-core\/[\d\.]+\/browser\.min\.js/
```

Comment:
This is something I'll address in a future change to the dev example, by updating the webpack/server setup. For now this looks like a great workaround :+1: